microsoft-edge/webview2/get-started/hololens2.md

```diff
@@ -63,17 +63,17 @@ You use the Unity game engine and Visual Studio to create your HoloLens 2 projec
 **Unity**, a versatile game engine, is a popular choice for creating 3D applications, including mixed reality experiences for HoloLens 2 devices, due to its extensive features and robust capabilities for augmented reality (AR), virtual reality (VR), and mixed reality (MR).
 
 
+<!-- ------------------------------ -->
+###### Mixed Reality Feature Tool (MRFT)
+
+The **Mixed Reality Feature Tool (MRTK)** is a new way for developers to discover, update, and add Mixed Reality feature packages into Unity projects. You can search packages by name or category, see their dependencies, and even view proposed changes to your projects manifest file before importing. If you've never worked with a manifest file before, it's a JSON file containing all your projects packages. Once you've validated the packages you want, the Mixed Reality Feature tool will download them into the project of your choice.
```

Comment:
```suggestion
The **Mixed Reality Feature Tool (MRTK)** is a new way for developers to discover, update, and add Mixed Reality feature packages into Unity projects. You can search packages by name or category, see their dependencies, and even view proposed changes to your projects manifest file before importing. A manifest file is a JSON file containing all your project's packages. Once you've validated the packages you want, the Mixed Reality Feature tool will download them into the project of your choice.
```

Reply:
"containing" is not precise.  "defining", "specifying"

Reply:
latest Files Changed tab shows this change so you can Resolve this comment thread